packages/expo-image/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "expo-image",
   "title": "Expo Image",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
Please, revert that. Changing the version is not needed, it'll be done by the publish script once we release it.

Reply:
Fixed under [1beedf1](https://github.com/expo/expo/pull/22491/commits/1beedf1628c3b6a5c606a0c7f06bc07ab0b30fe8).